mlrun/runtimes/__init__.py

```diff
@@ -96,6 +97,7 @@ class RuntimeKinds(object):
     serving = "serving"
     local = "local"
     handler = "handler"
+    databricks = "databricks-task"
```

Comment:
I think this might have already been discussed, but why `"databricks-task"` and not `"databricks"`, in line with other values?